Cargo.lock

```diff
@@ -4776,7 +4776,7 @@ dependencies = [
  "httpdate",
  "itoa",
  "pin-project-lite",
- "socket2 0.5.9",
+ "socket2 0.4.10",
```

Comment:
I've had `cargo` do this too where it flaps `socket2` back and forth between 0.5 and 0.4. It is... odd.